tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -347,6 +348,23 @@ def _setup_node(self, install_test_agent: bool) -> None:
         log.info("Copying %s to %s:%s", agent_package_path, self.context.node.name, target_path)
         self.context.ssh_client.copy_to_node(agent_package_path, target_path)
 
+        #
```

Comment:
it's probably be simpler to just retry the copy when the stderr of the first command contains "tar: Unexpected EOF in archive"